R/tm_g_distribution.R

```diff
@@ -1093,22 +1139,25 @@ srv_distribution <- function(id,
       qenv_final
     })
 
-
     dist_r <- reactive(dist_q()[["g"]])
 
     qq_r <- reactive(qq_q()[["g"]])
 
-    tests_r <- reactive(test_q()[["test_stats"]])
-
     output$summary_table <- DT::renderDataTable(
-      expr = common_q()[["summary_table"]],
+      expr = if (iv_r()$is_valid()) common_q()[["summary_table"]] else NULL,
       options = list(
         autoWidth = TRUE,
         columnDefs = list(list(width = "200px", targets = "_all"))
       ),
       rownames = FALSE
     )
 
+    tests_r <- reactive({
+      req(iv_r()$is_valid())
```

Comment:
![image](https://user-images.githubusercontent.com/15201933/210369863-f400919e-6226-44f7-9793-38f62ca25a39.png)

Reply:
#505

Reply:
This is actually NOT #505. Will be solved by #506